test/functional/ui/decorations_spec.lua

```diff
@@ -1596,6 +1596,7 @@ describe('decorations: inline virtual text', function()
       [18] = {background = Screen.colors.LightGrey, foreground = Screen.colors.Red};
       [19] = {background = Screen.colors.Yellow, foreground = Screen.colors.SlateBlue};
       [20] = {background = Screen.colors.LightGrey, foreground = Screen.colors.SlateBlue};
+      [21] = {reverse = true, foreground = Screen.colors.SlateBlue}
```

Comment:
This looks strange. What does this "reverse" come from?

Reply:
The first combine virtual text is in the "main" search selection which does "reverse" the highlighting of the text.